package/yast2-users.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Nov 30 20:40:58 UTC 2021 - David Diaz <dgonzalez@suse.com>
+
+- AutoYaST: do not warning about deprecated <group_password>
+  when found a blank one (related to jsc#SLE-20592).
```

Comment:
this is not precise. I would mention here explicitelly that we add "!" and "*" to ignored ones.

Reply:
Let's improve it